service/history/timerQueueActiveProcessor.go

```diff
@@ -120,7 +120,7 @@ func newTimerQueueActiveProcessor(
 	// if single cursor is enabled, then this processor is responsible for both active and standby tasks
 	// and we need to customize some parameters for ack manager and task executable
 	if singleProcessor {
-		timerTaskFilter = func(task tasks.Task) bool { return true }
+		timerTaskFilter = nil
```

Comment:
Multi-cursor already uses nil as the filter: https://github.com/temporalio/temporal/blob/master/service/history/queues/queue_base.go#L165